lib/service.ts

```diff
@@ -57,11 +56,11 @@ export class TranslateService {
 
 
   private _getParts(text: string) {
-    return pipe(
-      split('%s'),
-      map(split('%d')),
-      flattenArray
-    )(text);
+    return text.split('%s')
+      .map(function splitDelimiter(partialText) {
+        return partialText.split('%d')
+      })
+      .flat(Infinity)
```

Comment:
I think this should work:
`text.split(/%s|%d/gi)`

Reply:
done